parsec/core/gui/parsec_application.py

```diff
@@ -40,46 +43,46 @@ def event(self, e):
             # seemingly the only event we can use for this particular case,
             # ApplicationStateChange not being specific enough.
             self.get_main_window().show_top()
-        if e.type() != QEvent.FileOpen or e.url().scheme() != "parsec":
+        if e.type() != QEvent.FileOpen or e.url().scheme() != "parsec":  # type: ignore[attr-defined]
```

Comment:
`event` should probably be cast as a `QFileOpenEvent` when trying to access `url()` instead of using ignore.